fw-child/template/hierarchical-collapsible-menu.php

```diff
@@ -40,20 +40,32 @@
 
 ?>
 
-<div class="climatedata-menu">
+<div class="hierarchical-menu">
 	<?php
-        function climatedata_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
+        /**
+         * Displays the menu hierarchy and makes it collapsible.
+         *
+         * @param array $menu_structure The entire menu structure generated
+         * @param boolean $is_submenu Defines whether the current tree is a
+         *                            submenu or not
+         * @param int $submenu_id Optional. Ordinal to create unique ID for
+         *                        submenu. Default: null
+         * @param boolean $in_active_trail Optional. Defines if the current
+         *                                 menu is part of the active trail.
+         *                                 Default: null
+         */
+        function cdc_aside_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
```

Comment:
Maybe use "hierarchical" in the function's name:
```suggestion
        function cdc_hierarchical_menu( $menu_structure, $is_submenu, $submenu_id = NULL, $in_active_trail = NULL ) {
```

Reply:
@xfrenette-crim changed here and in the other places where it's needed.